src/js/components/DeploymentsListComponent.jsx

```diff
@@ -47,10 +47,19 @@ var DeploymentListComponent = React.createClass({
   },
 
   onRequestError: function (message, statusCode) {
+    var fetchState = States.STATE_ERROR;
+
+    switch (statusCode) {
+      case 401:
+        fetchState = States.STATE_UNAUTHORIZED;
+        break;
+      case 403:
+        fetchState = States.STATE_FORBIDDEN;
+        break;
```

Comment:
Absolutely, I am thinking about a more spread refactoring to this behavior in another PR.